device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/port_config.ini

```diff
@@ -1,57 +1,57 @@
+Ethernet0       0,1                    50000        etp1a       0
+Ethernet2       2,3                    50000        etp1b       0
+Ethernet4       4,5                    50000        etp2a       1
+Ethernet6       6,7                    50000        etp2b       1
+Ethernet8       8,9                    50000        etp3a       2
+Ethernet10      10,11                  50000        etp3b       2
+Ethernet12      12,13                  50000        etp4a       3
+Ethernet14      14,15                  50000        etp4b       3
+Ethernet16      16,17                  50000        etp5a       4
+Ethernet18      18,19                  50000        etp5b       4
+Ethernet20      20,21                  50000        etp6a       5
+Ethernet22      22,23                  50000        etp6b       5
+Ethernet24      24,25,26,27            100000       etp7        6
+Ethernet28      28,29,30,31            100000       etp8        7
+Ethernet32      32,33,34,35            100000       etp9        8
+Ethernet36      36,37,38,39            100000       etp10       9
+Ethernet40      40,41                  50000        etp11a      10
+Ethernet42      42,43                  50000        etp11b      10
+Ethernet44      44,45                  50000        etp12a      11
+Ethernet46      46,47                  50000        etp12b      11
+Ethernet48      48,49                  50000        etp13a      12
+Ethernet50      50,51                  50000        etp13b      12
+Ethernet52      52,53                  50000        etp14a      13
+Ethernet54      54,55                  50000        etp14b      13
+Ethernet56      56,57                  50000        etp15a      14
+Ethernet58      58,59                  50000        etp15b      14
+Ethernet60      60,61                  50000        etp16a      15
+Ethernet62      62,63                  50000        etp16b      15
+Ethernet64      64,65                  50000        etp17a      16
+Ethernet66      66,67                  50000        etp17b      16
+Ethernet68      68,69                  50000        etp18a      17
+Ethernet70      70,71                  50000        etp18b      17
+Ethernet72      72,73                  50000        etp19a      18
+Ethernet74      74,75                  50000        etp19b      18
+Ethernet76      76,77                  50000        etp20a      19
+Ethernet78      78,79                  50000        etp20b      19
+Ethernet80      80,81                  50000        etp21a      20
+Ethernet82      82,83                  50000        etp21b      20
+Ethernet84      84,85                  50000        etp22a      21
+Ethernet86      86,87                  50000        etp22b      21
+Ethernet88      88,89,90,91           100000        etp23       22
+Ethernet92      92,93,94,95           100000        etp24       23
+Ethernet96      96,97,98,99           100000        etp25       24
+Ethernet100     100,101,102,103        50000        etp26       25
```

Comment:
Speed and lane info doesn't match here. Is it a typo?

Reply:
Nice catch.
Fixed.